copyright_and_licenses.csv

```diff
@@ -19,6 +19,7 @@ au_icsm_GDA94_GDA2020_conformal.tif,Intergovernmental Committee on Surveying and
 au_icsm_National_84_02_07_01.tif,Intergovernmental Committee on Surveying and Mapping (ICSM),CC-BY-4.0,,
 au_icsm_README.txt,Disclaimed,Public domain,,
 be_ign_bd72lb72_etrs89lb08.tif,Insititut Geographique National (IGN) Belgium,CC-BY-4.0,,
+be_ign_hBG18.tif,Insititut Geographique National (IGN) Belgium,CC-BY-4.0,1.9,
```

Comment:
Minor typo here: `Insititut` should be `Institut`

Reply:
And `Geographique` should have an accent in both lines: `Géographique`, given that the French name is used :wink:

Reply:
fixed